go1.15/base-arm/Dockerfile.tmpl

```diff
@@ -0,0 +1,68 @@
+ARG DEBIAN_VERSION
+FROM arm64v8/debian:${DEBIAN_VERSION}
+
+ARG DEBIAN_VERSION
+# Replace sources.list in order to use archive.debian.org.
+COPY sources-debian${DEBIAN_VERSION}.list /etc/apt/sources.list
+
+RUN \
+    apt-get -o Acquire::Check-Valid-Until=false update \
+        && apt-get dist-upgrade -y \
+        && apt-get install -qq -y --no-install-recommends --allow-unauthenticated \
+            build-essential \
+            ca-certificates \
+            curl \
+            git \
+            gnupg \
+            make \
+            file \
+            flex \
+            bison \
+            librpm-dev \
+            libc-dev \
+            libpopt-dev \
+            linux-libc-dev \
+            libxml2-dev \
+            libxml2 \
+            libicu-dev \
+            libicu57 \
+            icu-devtools \
+            libsystemd-dev \
+            librpm3  \
+            librpmio3 \
+            librpmbuild3 \
+            librpmsign3 \
+            libsqlite3-dev \
+            libnss3 \
+            libsqlite3-0 \
+        && rm -rf /var/lib/apt/lists/*
+
+ARG GOLANG_VERSION=1.15.7
+ARG GOLANG_DOWNLOAD_URL=https://golang.org/dl/go$GOLANG_VERSION.linux-arm64.tar.gz
+ARG GOLANG_DOWNLOAD_SHA256=bca4af0c20f86521dfabf3b39fa2f1ceeeb11cebf7e90bdf1de2618c40628539
+
+RUN curl -fsSL "$GOLANG_DOWNLOAD_URL" -o golang.tar.gz \
+	&& echo "$GOLANG_DOWNLOAD_SHA256  golang.tar.gz" | sha256sum -c - \
+	&& tar -C /usr/local -xzf golang.tar.gz \
+	&& rm golang.tar.gz
+
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+WORKDIR $GOPATH
+
+COPY rootfs /
+
+# show the GLIBC version
+RUN ldd --version
+
+WORKDIR /
+RUN go get -d . \
+    && go build -o /crossbuild /entrypoint.go \
+    && rm -rf /go/* /root/.cache/* /entrypoint.go
+
+ENV GOLANG_CROSSBUILD=0
```

Comment:
This should be set to 1.

Reply:
ok, it is a flag on beats https://github.com/elastic/beats/blob/master/dev-tools/mage/build.go#L89

Reply:
```suggestion
ENV GOLANG_CROSSBUILD=1
```